server/routes/getting_started/getting_started_router.ts

```diff
@@ -134,4 +129,67 @@ export function registerGettingStartedRoutes(router: IRouter) {
       }
     }
   );
+
+  router.post(
+    {
+      path: `/api/observability/gettingStarted/mdsLoading`,
```

Comment:
May be can update the name:
From `/api/observability/gettingStarted/mdsLoading`
To `/api/observability/gettingStarted/createAssets`

Reply:
Done